C/Coin-OR/Osi/build_tarballs.jl

```diff
@@ -58,9 +67,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("CoinUtils_jll", compat="$(CoinUtils_version)"),
-    Dependency("OpenBLAS32_jll", OpenBLAS32_version),
-    Dependency("CompilerSupportLibraries_jll"),
+    Dependency(PackageSpec(name="CoinUtils_jll", uuid="be027038-0da8-5614-b30d-e42594cb92df"), compat="$(CoinUtils_version)"),
```

Comment:
```suggestion
    Dependency(PackageSpec(name="CoinUtils_jll", uuid="be027038-0da8-5614-b30d-e42594cb92df"), compat="=$(CoinUtils_version)"),
```

Reply:
We need to ensure strict compat `=` for all COIN-OR stuff going forward.

Reply:
Ok, let's do like Ipopt.
In theory we don't need the strict compat `=` but as we observed in the past, updating one dependency could break all COIN-OR packages.